lblod/ember-rdfa-editor-lblod-plugins
# GN-4539: Rename references to `decision`

### Overview

GN-4539: Rename references to `decision`

Instead of using `decision` use `legalDocument`, to not confuse with `besluit` usage (which also means "decision" in Flemish)

GN-4539: Rename `besluit` to `gemeentebesluit` for the citation plugin dropdown


#### Connected issues and PRs:

https://binnenland.atlassian.net/browse/GN-4539


### Setup

1. Checkout
2. `npm run start`

### How to test/reproduce

1. Open Besluit test page
2. See if the Citation plugin still works - search, insert, view details, inline search (using "gemeentebesluit")


### Checks PR readiness
